tests/cmsis_rtos_v1/src/thread_apis.c

```diff
@@ -10,6 +10,8 @@
 
 #define STACKSZ		512
 
+static osPriority osPriorityDeadline = 10;
```

Comment:
osPriority enum already provides a osPriorityError for non existing priority.
I think it would good to use it, instead of hard coding a random priority number.

Reply:
Yeah I have checked `osPriorityError` and though the result of check would be same, this is why I thought it is inappropriate to use:
It is set when

> system cannot determine priority or thread has illegal priority

Here we are checking if some one sets one of the priorities used in real-time conditions, as opposed to setting it to a pre-existing error value.